.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+repos:
+    - repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v2.3.0
+      hooks:
+          - id: check-yaml
+          - id: end-of-file-fixer
+          - id: trailing-whitespace
+          - id: check-ast
+          - id: check-byte-order-marker
+          - id: check-xml
+    - repo: https://github.com/psf/black
+      rev: 19.3b0
```

Comment:
A detail, but please change this to the latest 19.10b0 as is in the pyproject.toml: https://github.com/epidemics/epimodel/blob/master/pyproject.toml#L33 . I noticed that my PRs generates diffs because you are using old version which is not in sync in pyproject.toml @gavento

Reply:
:facepalm: Thanks!